src/app/components/login/login.component.ts

```diff
@@ -23,36 +27,30 @@ export class LoginComponent {
     type: '',
     text: ''
   }
-  constructor(private FormBuilder: FormBuilder, private authService: AuthService, private router: Router) {
+  constructor(private FormBuilder: FormBuilder, private authService: AuthService, private router: Router, private store: Store) {
      this.loginForm = this.FormBuilder.group({
       email: this.email,
       password: this.password
      })
   }
 
+
+  ngOnInit(): void {
+    this.store.select(selectToken).subscribe(token => {
```

Comment:
No this is not the way to goa bout it

Reply:
another approach has been used.
Please review and leave comment